dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.33/V2_33_36__Add_translations_column_into_programruleaction_table.sql

```diff
@@ -0,0 +1 @@
+alter table programruleaction add column if not exists translations jsonb;
```

Comment:
This is a script for 2.33, and not 2.34, so it should be fixed on the 2.33 branch, not 2.34. Or is this a relevant problem for 2.34?

I would in either case suggest removing this file from this PR, and fixing it in a separate PR